waggle/plugin.py

```diff
@@ -57,11 +57,12 @@ def time_ns():
 
 
 class Message(NamedTuple):
+    vers: str
```

Comment:
Same as the vers comment before - version is more about the version of the protocol and not the message content itself, so we can cut this field (or make it optional).